packages/flutter/test/animation/tween_test.dart

```diff
@@ -49,4 +52,28 @@ void main() {
     expect(tween.lerp(0.5), equals(Rect.lerp(a, b, 0.5)));
     expect(tween, hasOneLineDescription);
   });
+
+  test('Matrix4Tween', () {
+    final Matrix4 a = new Matrix4.identity();
+    final Matrix4 b = new Matrix4.identity()
+      ..translate(6.0, -8.0, 0.0)
+      ..scale(0.5, 1.0, 5.0);
+    final Matrix4Tween tween = new Matrix4Tween(begin: a, end: b);
+    expect(tween.lerp(0.0), equals(a));
+    expect(tween.lerp(1.0), equals(b));
+    expect(
```

Comment:
flutter style is that if you have a newline after the `(`, you should have one before the matching `)`, and the contents should be indented by 2.